crates/core/src/messages/control_db.rs

```diff
@@ -36,6 +36,15 @@ pub struct Database {
     ///
     /// Updating the database's module will **not** change this value.
     pub initial_program: Hash,
+
+    /// Replication configuration for the database.
+    pub replication: ReplicationConfig,
+}
+
+#[derive(Clone, Default, PartialEq, Serialize, Deserialize)]
+pub struct ReplicationConfig {
```

Comment:
I might be missing something, but I think we can omit this here -- we only need the type from the control _module_.